<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1386</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1386</width>
    <height>636</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1386</width>
    <height>636</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UAR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="zadana" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>781</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="uchyb" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sterowanie" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>280</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="amplitudaLabel">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>590</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>560</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>587</x>
      <y>590</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>amplituda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>560</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>td</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delayLabel">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>590</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="okresLabel">
    <property name="geometry">
     <rect>
      <x>796</x>
      <y>560</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zakloceniaLabel">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>560</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cyklLabel">
    <property name="geometry">
     <rect>
      <x>796</x>
      <y>590</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>590</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>587</x>
      <y>560</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>zaklocenia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>560</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>typ sygnalu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vecbLabel">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>560</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>590</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cykl</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>590</y>
      <width>28</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>560</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tdLabel">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>560</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>590</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ti</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiLabel">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>590</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kpLabel">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>560</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="vecaLabel">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>590</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>590</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>czas aktywacji</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>560</y>
      <width>28</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>okres</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sygnalcomboBox">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>560</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aktywacjaLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>590</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>180</y>
      <width>110</width>
      <height>214</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Wejście</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="zadaneLabel"/>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Wyjście</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="wyjscieLabel"/>
       <widget class="QPushButton" name="aktualizujButton">
        <property name="text">
         <string>AKTUALIZUJ</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
